### DISCUSSION TITLE
请问配置文件中哪些是必须被插件接管的项？

### DISCUSSION THREAD

**Discussion by liumingan** [nikkinikki-org/OpenWrt-nikki#114] [Q&A]
或者是说即使关闭覆写，插件也必然会接管哪些配置文件中的项目

**Comment by morytyann** [comment]
大哥你看看[Wiki](https://github.com/morytyann/OpenWrt-mihomo/wiki)啊

**Reply by liumingan** [reply]
我看了wiki，因为我是关闭覆写的，配置文件中的嗅探是打开的，然而在启动日志里发现覆写是关闭，所以才到此一问
